backend/src/main/java/gov/cdc/usds/simplereport/utils/BulkUploadResultsToFhir.java

```diff
@@ -415,20 +416,22 @@ private Bundle convertRowToFhirBundle(TestResultRow row, UUID orgId) {
                 .receivedTime(testingLabSpecimenReceivedDate)
                 .build());
 
+    String testResultSnomed = getTestResultSnomed(row.getTestResult().getValue());
+    SnomedConceptRecord resultConceptRecord = Translators.getSnomedConceptByCode(testResultSnomed);
+
     var resultObservation =
         List.of(
             fhirConverter.convertToObservation(
                 ConvertToObservationProps.builder()
                     .testPerformedLoinc(row.getTestPerformedCode().getValue())
                     .diseaseName(diseaseName)
-                    .resultCode(getTestResultSnomed(row.getTestResult().getValue()))
+                    .resultCode(getTestResultSnomed(testResultSnomed))
```

Comment:
Should this just be `.resultCode(testResultSnomed)` since that variable is already the result of `getTestResultSnomed`?

Reply:
Yes!  Good catch